tox.ini

```diff
@@ -10,8 +10,12 @@ deps =
     pytest-coverage
     mypy
 
+setenv =
+    VIRTUOSO=1
+
 commands =
-    black rodlayout --line-length 100 --skip-string-normalization --check
-    flake8 --max-line-length 100 rodlayout
+    black rodlayout tests --line-length 100 --skip-string-normalization --check
+    flake8 --ignore=E203,W503 --max-line-length 100 rodlayout tests
```

Comment:
W503 doesn't even happen, E203 can be fixed.

Maybe write a `bbox_to_rect(bbox)` function
```
(left, right), (bottom, top) = bbox
return Rect.from_edges(left, right, bottom, top)
```

Reply:
bbox_to_rect solves E203 in this case. W503 appeared as a consequence to ignoring E203. However, https://unihd-cag.github.io/simple-geometry/ documents the usage of Rect[], which may lead to black and flake8 disagreeing. I suggest to keep the ignore flag, as is recommended by black.

See https://github.com/psf/black#line-breaks--binary-operators and https://github.com/psf/black#slices regarding compatibility of black and flake8 regarding E203 and W503.

Reply:
The ignore should be left out since it hints towards a too complex line of code